plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -415,6 +422,210 @@ createWidget("discourse-poll-info", {
   }
 });
 
+function transformUserFieldToLabel(fieldName) {
+  let transformed = fieldName.split("_").filter(Boolean);
+  transformed[0] =
+    transformed[0].charAt(0).toUpperCase() + transformed[0].slice(1);
+  return transformed.join(" ");
+}
+
+createWidget("discourse-poll-grouped-pies", {
+  tagName: "div.poll-grouped-pies",
+  buildAttributes(attrs) {
+    return {
+      id: `poll-results-grouped-pie-charts-${attrs.id}`
+    };
+  },
+
+  html(attrs) {
+    const fields = Object.assign({}, attrs.groupableUserFields);
+    attrs.groupedBy = attrs.groupedBy || fields[0];
+    const fieldSelectId = `field-select-${attrs.id}`;
+
+    let contents = [];
+
+    let selectInnerHtml = "";
+    attrs.groupableUserFields.forEach(field => {
+      selectInnerHtml += `<option value='${field}' ${
+        attrs.groupedBy === field ? "selected" : ""
+      } >${transformUserFieldToLabel(field)}</option>`;
+    });
+    contents.push(
```

Comment:
Is there a reason you are not pushing HTML elements using the `h` helper? `RawHtml` is slower and should be avoided if you can.